src/style.css

```diff
@@ -8,14 +8,14 @@ div {
   width: 100%;
 }
 
-.red {
+.red-box {
```

Comment:
The theme of the site may be changed and binding class names to colors is bad practice. You can think of these blocks as content, wrapper, and inner wrapper. Example: outer, inner, wrapper, content, etc.

Reply:
fixed